ui/app/styles/components/policies.scss

```diff
@@ -25,3 +25,48 @@ table.policies {
     margin-bottom: 1rem;
   }
 }
+
+.token-operations {
+  margin-bottom: 3rem;
+  display: grid;
+  grid-auto-flow: column;
+  grid-template-columns: repeat(auto-fit, 50%);
+  grid-auto-rows: minmax(100px, auto);
+  gap: 1rem;
+
+  .boxed-section {
+    padding: 0;
+    margin: 0;
+    display: grid;
+    grid-template-rows: auto 1fr;
+
+    .external-link svg {
+      position: relative;
+      top: 3px;  
+    }
+
+    button.create-test-token, pre {
+      margin-top: 1rem;
+    }
+
+    pre {
+      display: grid;
+      grid-template-columns: 1fr auto;
+      align-content: flex-start;
+      white-space: normal;
+      overflow: visible;
+    }
+  }
+}
+
+@media #{$mq-hidden-gutter} {
+  .token-operations {
+    grid-auto-flow: row;
+    grid-template-columns: 1fr;
+  }
+}
```

Comment:
De-columnizes things if you're on a narrow screen
![image](https://user-images.githubusercontent.com/713991/206575283-d85e34aa-6de1-4e32-ad8e-f68e5ff3b8b4.png)